sourmash_lib/sbt.py

```diff
@@ -133,6 +126,12 @@ def add_node(self, node):
         elif isinstance(p.node, Node):
             self.nodes[pos] = node
             node.update(p.node)
+        elif p.node is None:
+            n = Node(self.factory, name="internal." + str(p.pos))
+            self.nodes[p.pos] = n
+            c1 = self.children(p.pos)[0]
+            self.nodes[c1.pos] = node
+            node.update(n)
```

Comment:
I saw this happen when using bigger SBTs, I'll try to cook up a test without putting too much data in the repo